config/initializers/carrierwave.rb

```diff
@@ -31,6 +32,8 @@
     }
     config.fog_directory = Barong::App.config.storage_bucket_name
   elsif 'AliCloud'.casecmp?(Barong::App.config.storage_provider)
+    Barong::App.set(:uploader, AliUploader)
```

Comment:
So we reassign it even if env was another?
We should consider removing env and making it inside this if elsif